application-example.yml

```diff
@@ -1,9 +1,11 @@
 twitter:
-  - userName:
-    consumerKey:
-    consumerSecret:
-    accessToken:
-    accessSecret:
+  defaultuser: 0
+  users:
+    - userName:
```

Comment:
come up with a better variable name for what this value actual represents, variable name is confusing